schunk_gripper_library/schunk_gripper_library/driver.py

```diff
@@ -378,6 +378,8 @@ def start() -> bool:
             return self.wait_for_status(bits=desired_bits, timeout_sec=0.1)
 
         def check() -> bool:
+            if self.get_status_bit(11):
```

Comment:
@PrajwalMangaluru 
Why this explicit check here for bit 11 (no workpiece detected) and what would consequences be?

Reply:
Makes sense. Should we include that into the `desired_bits`  with `"11": 0"` or does it need to be separate?